### ISSUE TITLE
Save all comments when saving responses for all questions

### ISSUE THREAD

**Issue by zhaojj2209** [TEAMMATES/teammates#12527]
Currently, the "Submit Responses to All Questions" button does not save comments. Given that comments for responses are now allowed for all question types (see #12426), it is all the more important that comments are saved along with the responses.

**Comment by Saimurugeshwari** [comment]
can I work on this issue.

**Comment by jasonqiu212** [comment]
Hi @Saimurugeshwari, sure! Feel free to submit a PR for this issue

**Comment by ellaella12** [comment]
Hi @Saimurugeshwari, may i check if you are working on this issue? if not, i would like to give it a go!

**Comment by Saimurugeshwari** [comment]
Hi @ellaella12 , I am still working on this issue, Kind of stuck, I will raise a PR request today or tomorrow, Sorry for the delay.

**Comment by dlimyy** [comment]
Hi, may I check if this issue is currently still being worked on by anyone as I would like to give this issue a try. Thanks!

**Comment by dlimyy** [comment]
Hi, I am currently working on this issue and am experiencing some difficulties in replicating the error identified in the issue. I have attempted to click the "Submit Responses to All Questions" button but noticed that the comments were saved in my end instead of being unsaved as stated in the issue(I checked from the instructor's perspective(through view results) and student's perspective). I am wondering if anyone knows the steps that can be taken to replicate the bug. Thanks!
